modules/ravestate_akinator/__init__.py

```diff
@@ -148,21 +141,34 @@ def is_it_answered(ctx):
             out = "Yeah! I guessed right! Thanks for playing with me! \nDo you want to play again?"
         elif ctx["nlp:yesno"] == "no":
             akinator_api.exclusion_get_request()
-            out = "I guessed wrong but do you want to play again?"
+            out = "Oh no! I guessed wrong but do you want to play again?"
         else:
-            # TODO catch wrong input for this state
             out = "What? But do you want to play again?"
         ctx["rawio:out"] = out
         ctx["akinator:initiate_play_again"] = True
 
 
-    @state(cond=s("akinator:wrong_input:changed"),
-           write=("rawio:out", "akinator:question"),
-           emit_detached=True)
+    @state(cond=s("akinator:wrong-input"),
+           write="rawio:out",
+           emit_detached=True,
+           signal_name="exit-game")
     def wrong_input(ctx):
         """
         Catches wrong inputs from the interlocutor during questions answering and loops back to the question state
         """
-        ctx["rawio:out"] = "Sadly I could not process that answer. Remember that you have these five answering choices: " \
+        ctx["rawio:out"] = "Sadly I could not process that answer. Do you want to stop playing Akinator?"
+        return Emit()
+
+
+    @state(cond=s("nlp:yesno:changed") & s("akinator:exit-game", max_age=13.),
+           write=("rawio:out", "akinator:question"),
+           read="nlp:yesno",
+           emit_detached=True)
+    def exit_game_answered(ctx):
+        if ctx["nlp:yesno"] == "yes":
+            ctx["rawio:out"] = "Thanks for playing with me!"
+            return Resign()
+        else:
+            ctx["rawio:out"] = "Yeah! Let's keep playing! Remember that you have these five answering choices:" \
```

Comment:
same as before, maybe adapt a little bit for better synthesis

Reply:
:)